ramls/instance.json

```diff
@@ -325,6 +325,26 @@
         ]
       }
     },
+    "instanceDate": {
+      "type": "object",
+      "description": "Instance Date object contains InstanceDateType ID and Date1/Date2 which based on the 008/06 position",
+      "properties": {
+        "instanceDateTypeId": {
+          "type": "string",
+          "description": "UUID for the instance date type",
+          "$ref": "uuid.json"
+        },
```

Comment:
```suggestion
    "dates": {
      "type": "object",
      "description": "Instance Dates",
      "properties": {
        "dateTypeId": {
          "type": "string",
          "description": "Date type ID",
          "$ref": "uuid.json"
        },
```

Reply:
1. instance does not represent MARC only, it could be instance without MARC source but still these dates could be used. so any mentions about marc must be removed.
2. PO suggested to have `dates` name for the object